### ISSUE TITLE
East Prussian Offensive Cannot Move Units Once Placed

### ISSUE THREAD

**Issue by metromoran** [nicupavel/openpanzer#130]
I really appreciate you creating this game. I played the PanzerBlitz board game as a kid and really enjoyed it. 

I'm stuck on the East Prussian Offensive from the The Great Patriotic War. I can place my units and move them off the deployment hexes, but cannot move them after that. I've attached a zipped .json file for you to look at if you have a chance.

[(Campaign) East Prussian Offensive Turn 5 Mon Oct 23 2017 2-20-47 AM.zip](https://github.com/nicupavel/openpanzer/files/1406128/Campaign.East.Prussian.Offensive.Turn.5.Mon.Oct.23.2017.2-20-47.AM.zip)

**Comment by salexgit** [comment]
Hi

Which version did you use? The latest beta has a number of fixes which might help...

**Comment by salexgit** [comment]
I opened your saved game with beta, and I have the same problem, I cannot move any unit. Did this problem occur at turn#5? If you have the saved game at turn#1, can you replay and confirm that the problem occurs again at turn#5? 

I saw some weird problems with the game, which sometimes can be fixed/circumvented around by replaying or by using a different browser.

**Comment by metromoran** [comment]
I think it occurs at the very beginning of the game or right at turn#1. I
can place units in the deployment hex and move them out, but when I click
the Finish Turn button I can tell that something is up. The "Computer Turn
in Progress" dropdown window does not display, but the turn does end and a
new one begin. There were no problems in the previous scenario Budapest
Offensive. I've attached my save from the very start of the game.

Sean

**Comment by salexgit** [comment]
Did you attach the game at turn#1? The filename you sent says "...Turn 5...". Do you have a saved game before the problem occurs, i.e. right in the beginning when the units are not deployed yet and/or previous scenario before the last victory hex is captured.

Just out of curioisity I started this scenario in a standalone mode, but I was not able to reproduce the problem. I was able to deploy units and move them around.

**Comment by metromoran** [comment]
I thought I attached it to the previous email, but maybe I need to go back
to GitHub and post it there. I'll do that. So can I start openpanzer/beta
in standalone mode?

Sean

**Comment by metromoran** [comment]
Here a zip with Turn#1 for the East Prussian Offensive. Sean

[(Campaign) East Prussian Offensive Turn 1 Wed Oct 18 2017 3-29-18 AM.zip](https://github.com/nicupavel/openpanzer/files/1409299/Campaign.East.Prussian.Offensive.Turn.1.Wed.Oct.18.2017.3-29-18.AM.zip)

**Comment by metromoran** [comment]
reopen (accidentally closed)

**Comment by salexgit** [comment]
Regarding your question on the offline/standalone mode, do please check #131. It contains information on where you can get the offline package from.

**Comment by salexgit** [comment]
Coming back to the problem.... I experimented a bit with the saved game and it seems that the game engine simply does not move to the next turn. When you press the "next turn" button, there is no "Computer turn in progress" notification, i.e. the game literally freezes. Interestingly enough, if you start the game as a separate scenario, then everything works fine. 

Do you have a saved campaign game at the end of the previous scenario?

**Comment by metromoran** [comment]
Hi...got a little busy with work. I've attached a saved game from the previous scenario that worked fine. Thanks for looking at this.

[(Campaign) Budapest Offensive Turn 21 Wed Oct 18 2017 3-17-50 AM.zip](https://github.com/nicupavel/openpanzer/files/1416477/Campaign.Budapest.Offensive.Turn.21.Wed.Oct.18.2017.3-17-50.AM.zip)

**Comment by salexgit** [comment]
Hi... Do you have a "Budapest Offensive" saved game at the last turn before capturing the last victory hex? Since the standalone non-campaign scenario works fine, I want to test whether this problem occurs only for a specific saved game or whether it is a systematic error for any saved game of this campaign.

**Comment by metromoran** [comment]
I don't have a Budapest Offensive saved game for the last turn, just up to Turn 21. I'll try to play it up to the end and save it when I get a chance. I also was able to start moving the unit though using the attached save for East Prussian Offensive. Once I moved the infantry units to the west and engaged the German units and then ended the turn, I could then move the other units as well. Weird. 

[(Campaign) East Prussian Offensive Turn 1 Wed Oct 18 2017 11-25-06 PM.zip](https://github.com/nicupavel/openpanzer/files/1418313/Campaign.East.Prussian.Offensive.Turn.1.Wed.Oct.18.2017.11-25-06.PM.zip)
